src/app/FakeLib/CscHelper.fs

```diff
@@ -78,11 +78,22 @@ let cscExe toolPath (srcFiles : string list) (opts : string list) : int =
 ///                                Target = ...
 ///                                ... })
 let csc (setParams : CscParams -> CscParams) (inputFiles : string list) : int =
-    let inputFiles = inputFiles |> Seq.toList
+    // Helper to quote a path with spaces in it, if not already quoted.  See https://github.com/fsharp/FAKE/issues/992
+    let ensureTrimQuotedPath (path : string) =
```

Comment:
Can we put this in one of the other helpers? Maybe we already have something similar